src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -219,6 +219,17 @@ internal bool IsDNADataIngestionPipelineinBeta
             }
         }
 
+        /// <summary>
+        /// Controls if the cluster node autocomplete placement feature is enabled from feature flag
+        /// </summary>
+        internal bool IsNodeAutocompleteClusterEnabled
+        {
+            get
+            {
+                return DynamoModel.FeatureFlags?.CheckFeatureFlag("IsNodeAutocompleteClusterEnabled", false) ?? true;
```

Comment:
@reddyashish I found we already have one feature flag for it - `IsDNAClusterPlacementEnabled`, default value is on

Reply:
updated it. Thanks Aaron